source/dotnet/wholesale-integration-events-package/Contracts/Contracts.csproj

```
@@ -95,6 +95,14 @@ limitations under the License.
       <CompileOutputs>True</CompileOutputs>
       <Generator>MSBuild:Compile</Generator>
     </Protobuf>
+    <Protobuf Include="../../wholesale-api/Events/Events.Infrastructure/IntegrationEvents/GridLossResultProducedV1/Protos/*.proto">
+      <GrpcServices>None</GrpcServices>
+      <Access>Public</Access>
+      <ProtoCompile>True</ProtoCompile>
+      <ProtoRoot>../../wholesale-api/Events/Events.Infrastructure/IntegrationEvents</ProtoRoot>
+      <CompileOutputs>True</CompileOutputs>
+      <Generator>MSBuild:Compile</Generator>
+    </Protobuf>
```

Instead of keep adding, could we do something like this?
```suggestion
    <Protobuf Include="../../wholesale-api/Events/Events.Infrastructure/IntegrationEvents/**/*.proto">
      <GrpcServices>None</GrpcServices>
      <Access>Public</Access>
      <ProtoCompile>True</ProtoCompile>
      <ProtoRoot>../../wholesale-api/Events/Events.Infrastructure/IntegrationEvents</ProtoRoot>
      <CompileOutputs>True</CompileOutputs>
      <Generator>MSBuild:Compile</Generator>
    </Protobuf>
```